db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_03_12_211214) do
+ActiveRecord::Schema.define(version: 2021_03_15_162119) do
```

Comment:
As mentioned above, this will need to be in a separate PR with the migration.

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/6176